AVsitter2/Plugins/AVprop/[AV]menu.lsl

```diff
@@ -627,6 +628,17 @@ default
                 {
                     MTYPE = (integer)part0;
                 }
+				else if (command == "LMSOURCE")
+				{
+					if(llToLower(part0) == "self")
```

Comment:
I'd prefer numeric codes, rather than keywords, for consistency with the other commands and to save memory. The latter is the reason why MTYPE etc. are numeric.